### PR TITLE
Search card subcategories 

### PR DESCRIPTION
**Pull request by adriencyberspace**
# Pull Request Template

## Issue Overview

This PR addresses [Add subcategory tags to individual service cards](https://www.notion.so/exygy/Add-subcategory-tags-to-individual-service-cards-54fe65fc25ca496fa883f2b06d4aa8c7)

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR adds the subcategory tags to the search page listing cards, plus styling changes needed to accommodate designs on the card. NOTE: The labels don't all match the eligibility filters. That is out of this scope, and likely out of MVP scope.

## How Can This Be Tested/Reviewed?

Visit any category listings page or search something in the search bar to get to /search page. The cards will have subcategory labels per mockup, as well as a tooltip that shows the rest of the subcategories.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have assigned reviewers
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: app/components/search/SearchResults/SearchResults.module.scss [hunk 1]
```diff
@@ -129,18 +147,34 @@
...
-    grid-template-columns: none;
-    justify-items: start;
+  @media screen and (max-width: $break-tablet-s) {
+    margin-top: $general-spacing-s;
   }
 }
 
+.searchResultSubcat,
+.searchResultSubcatTooltip {
+  border-radius: var(--rounded-full, 9999px);
```
Comment (rosschapman) [inline app/components/search/SearchResults/SearchResults.module.scss:162]: Oh, interesting hack. Just read about it.

File: app/styles/utils/_helpers.scss [hunk 2]
```diff
@@ -59,7 +59,7 @@ table {
 @import url(https://fonts.googleapis.com/icon?family=Material+Icons);
 @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,700&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400..700&display=swap");
```
Comment (rosschapman) [inline app/styles/utils/_helpers.scss:62]: Oh, was this a bug?

**Review by rosschapman — APPROVED** [review]

File: app/styles/utils/_helpers.scss (hunk 2 above)
Comment (adriencyberspace) [reply app/styles/utils/_helpers.scss:62]: No bug, i just want 500 and 600 too

File: app/components/search/SearchResults/SearchResults.module.scss (hunk 1 above)
Comment (adriencyberspace) [reply app/components/search/SearchResults/SearchResults.module.scss:162]: oops actually, I should change this to a var. I copied and pasted from figma.

File: app/styles/utils/_helpers.scss (hunk 2 above)
Comment (rosschapman) [reply app/styles/utils/_helpers.scss:62]: Nice, gtk. 👍🏻

**Pull request merged**